src/field_spec.c

```diff
@@ -8,12 +8,10 @@ RSValueType fieldTypeToValueType(FieldType ft) {
       return RSValue_Number;
     case INDEXFLD_T_FULLTEXT:
     case INDEXFLD_T_TAG:
-      return RSValue_String;
     case INDEXFLD_T_GEO:
-    default:
-      // geo is not sortable so we don't care as of now...
-      return RSValue_Null;
+      return RSValue_String;
   }
+  return RSValue_Null;
```

Comment:
Maybe add as `default:` to the switch?

Reply:
We usually avoid `case default:` so we get a warning when a new enum is introduced (thank you for teaching me that). 
The `return` statement at the end is to silence the compiler.